j5/backends/hardware/sr/v4/power_board.py

```diff
@@ -72,6 +73,8 @@ def discover(cls, find: Callable = usb.core.find) -> Set[Board]:
     def __init__(self, usb_device: usb.core.Device) -> None:
         self._usb_device = usb_device
 
+        self._lock = Lock()
```

Comment:
Since `RawUSBHardwareBackend` declares and uses `_lock`, could we move this to `RawUSBHardwareBackend.__init__`?

Reply:
b7bf497